qiskit/converters/ast_to_dag.py

```diff
@@ -26,8 +26,14 @@
 from qiskit.circuit.barrier import Barrier
 from qiskit.circuit.delay import Delay
 from qiskit.circuit.library import standard_gates as std
+from qiskit.utils.deprecation import deprecate_func
 
 
+@deprecate_func(
+    since="0.46",
+    additional_msg="Instead use the qiskit.qasm2.dump() or qiskit.qasm2.    dumps() function",
```

Comment:
Extra spaces here maybe?

Reply:
oh weird, how did this slip in.

```suggestion
    additional_msg="Instead use the qiskit.qasm2.dump() or qiskit.qasm2.dumps() function",
```